env.daint.sh

```diff
@@ -251,6 +251,9 @@ setFortranEnvironment()
         exit 1
     esac
 
+    # CLAW Compiler using the correct preprocessor
+    export CLAWFC="/project/c14/install/daint/claw_v1.1.0/${compiler}/bin/clawfc"
```

Comment:
you should use an env variable for /project/c14/install/daint/ otherwise this will break when we mode, there is one define in the machine.env

Reply:
Uses `${installdir}` now.